pkg/sdkclient/sideinput/interface.go

```diff
@@ -12,4 +12,6 @@ type Client interface {
 	CloseConn(ctx context.Context) error
 	IsReady(ctx context.Context, in *emptypb.Empty) (bool, error)
 	RetrieveSideInput(ctx context.Context, in *emptypb.Empty) (*sideinputpb.SideInputResponse, error)
+	WaitUntilReady(ctx context.Context) error
+	IsHealthy(ctx context.Context) error
```

Comment:
We don't necessarily need to add these 2 methods under the Client interface as they don't directly interact with gRPC. Consider keeping them as individual methods as we did for other SDK clients.